packages/create-invoice-form/src/lib/invoice/form.svelte

```diff
@@ -410,77 +411,65 @@
-            ?.filter((curr) => {
-              if (!curr) return false;
-              return (
-                curr.type === Types.RequestLogic.CURRENCY.ISO4217 ||
-                (curr.network && curr.network === network)
-              );
-            })
-            .map((currency) => ({
+
+        <div class="searchable-dropdown-container">
+          <SearchableDropdown
+            getValue={(currency) => currency.value.symbol}
+            getDisplayValue={(currency) =>
+              `${currency.value.symbol} ${currency.value.network ? `(${currency.value.network})` : ""}`}
+            placeholder="Invoice currency"
+            items={defaultCurrencies
+              ?.filter((curr) => {
+                if (!curr) return false;
+                return (
+                  curr.type === Types.RequestLogic.CURRENCY.ISO4217 ||
+                  (curr.network && curr.network === network)
+                );
+              })
+              .map((currency) => ({
+                value: currency,
+                label: `${currency?.symbol ?? "Unknown"} ${currency?.network ? `(${currency.network})` : ""}`,
+                type: "invoiceCurrency",
+              })) ?? []}
+            onSelect={handleInvoiceCurrencyChange}
+          />
+          <SearchableDropdown
+            items={networks
+              .filter((networkItem) => networkItem)
+              .map((networkItem) => {
+                return {
+                  value: networkItem,
+                  label: networkItem[0]?.toUpperCase() + networkItem?.slice(1),
+                  type: "network",
+                };
+              })}
+            placeholder="Payment chain"
+            getValue={(network) => network.value}
+            getDisplayValue={(network) => network.label}
+            onSelect={handleNetworkChange}
+          />
+          <SearchableDropdown
+            items={filteredSettlementCurrencies.map((currency) => ({
               value: currency,
-              label: `${currency?.symbol ?? "Unknown"} ${currency?.network ? `(${currency.network})` : ""}`,
-            })) ?? []}
-          onchange={handleInvoiceCurrencyChange}
-        />
-        <Dropdown
-          {config}
-          placeholder="Settlement currency"
-          selectedValue={currency
-            ? `${currency.symbol ?? "Unknown"} (${currency?.network ?? "Unknown"})`
-            : undefined}
-          options={filteredSettlementCurrencies.map((currency) => ({
-            value: currency,
-            label: `${currency.symbol ?? "Unknown"} (${currency?.network ?? "Unknown"})`,
-          }))}
-          onchange={handleCurrencyChange}
-        />
-        {#if cipherProvider}
-          <Input
-            type="checkbox"
-            id="isEncrypted"
-            label="Encrypt invoice"
-            bind:checked={formData.isEncrypted}
+              type: "settlementCurrency",
+            }))}
+            placeholder="Settlement currency"
```

Comment:
Please change this to "Payment currency" to match "Payment chain". :pray: 

```suggestion
            placeholder="Payment currency"
```

Reply:
Added 👍